### PR TITLE
[Android] Add bleed property support

### PR DESCRIPTION
**Pull request by almedina-ms**
Completes this work item: https://github.com/Microsoft/AdaptiveCards/issues/2360

### CODE REVIEW

File: source/android/adaptivecards/src/main/java/io/adaptivecards/renderer/readonly/ColumnRenderer.java [hunk 1]
```diff
@@ -42,6 +42,12 @@ public static ColumnRenderer getInstance()
         return s_instance;
     }
 
+    public void setInformationForColumnRendering(boolean isFirstColumn, boolean isLastColumn)
```
Comment (paulcam206) [inline source/android/adaptivecards/src/main/java/io/adaptivecards/renderer/readonly/ColumnRenderer.java]: >setInformationForColumnRendering [](start = 16, length = 32)

I think I'd prefer two methods here -- `setIsFirstColumn()` and `setIsLastColumn()`

File: source/android/adaptivecards/src/main/java/io/adaptivecards/renderer/readonly/ColumnRenderer.java [hunk 2]
```diff
@@ -42,6 +42,16 @@ public static ColumnRenderer getInstance()
...
     }
 
+    public void setIsRenderingFirstColumn(boolean isRenderingFirstColumn)
+    {
+        m_isRenderingFirstColumn = isRenderingFirstColumn;
+    }
+
+    public void setIsRenderingLastColumn(boolean isRenderingLastColumn)
+    {
+        m_isRenderingLastColumn = m_isRenderingLastColumn;
```
Comment (carlos-zamora) [inline source/android/adaptivecards/src/main/java/io/adaptivecards/renderer/readonly/ColumnRenderer.java]: >m_isRenderingLastColumn [](start = 34, length = 23)

Should this be `m_isRenderingLastColumn = isRenderingLastColumn`?

File: source/android/adaptivecards/src/main/java/io/adaptivecards/renderer/readonly/ColumnRenderer.java [hunk 3]
```diff
@@ -159,6 +174,29 @@ else if (columnSize.equals(g_columnSizeAuto))
             }
         }
 
+        if (column.GetBleed() && column.GetCanBleed())
+        {
+            long padding = Util.dpToPixels(context, hostConfig.GetSpacing().getPaddingSpacing());
+            int leftPadding = 0, rightPadding = 0;
+
+            if(m_isRenderingFirstColumn)
```
Comment (carlos-zamora) [inline source/android/adaptivecards/src/main/java/io/adaptivecards/renderer/readonly/ColumnRenderer.java]: formatting: space between `if` and `(`

File: source/android/adaptivecards/src/main/java/io/adaptivecards/renderer/readonly/ColumnRenderer.java [hunk 4]
```diff
@@ -159,6 +174,29 @@ else if (columnSize.equals(g_columnSizeAuto))
...
+            {
+                leftPadding = (int)-padding;
+                // Reset the flag just in case
+                setIsRenderingFirstColumn(false);
+            }
+
+            if(m_isRenderingLastColumn)
```
Comment (carlos-zamora) [inline source/android/adaptivecards/src/main/java/io/adaptivecards/renderer/readonly/ColumnRenderer.java]: formatting: space between `if` and `(`

File: source/android/adaptivecards/src/main/java/io/adaptivecards/renderer/readonly/ColumnSetRenderer.java [hunk 5]
```diff
@@ -80,13 +80,27 @@ else if ((columnSet = ColumnSet.dynamic_cast(baseCardElement)) == null)
...
         if(!baseCardElement.GetIsVisible())
         {
             layout.setVisibility(View.GONE);
         }
 
         for (int i = 0; i < columnVectorSize; i++) {
             Column column = columnVector.get(i);
+
+            ColumnRenderer rendererAsColumnRenderer = null;
+            if(columnRenderer instanceof ColumnRenderer)
```
Comment (carlos-zamora) [inline source/android/adaptivecards/src/main/java/io/adaptivecards/renderer/readonly/ColumnSetRenderer.java]: formatting: space between `if` and `(`

**Pull request merged**